package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "crypto-random-string",
-  "version": "1.0.0",
+  "version": "2.0.0",
```

Comment:
Don't bump version in a pull request. It's up to the maintainers when, how, and what version to bump.